tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -1,26 +1,120 @@
 *** Settings ***
-Resource    ../../../Resources/ODS.robot
+Resource            ../../../Resources/ODS.robot
+Test Setup          Begin Metrics Web Test
+Test Teardown       End Metrics Web Test
+
 
 *** Variables ***
+@{RECORD_GROUPS}    Availability Metrics    SLOs - JupyterHub    SLOs - ODH Dashboard    SLOs - RHODS Operator    SLOs - Traefik Proxy    Usage Metrics
 
-@{RECORD_GROUPS}  Availability Metrics  SLOs - JupyterHub  SLOs - ODH Dashboard  SLOs - RHODS Operator  SLOs - Traefik Proxy  Usage Metrics
+@{ALERT_GROUPS}     Builds                  DeadManSnitch    RHODS-PVC-Usage    SLOs-haproxy_backend_http_responses_total    SLOs-probe_success
 
-@{ALERT_GROUPS}  Builds  DeadManSnitch  RHODS-PVC-Usage  SLOs-haproxy_backend_http_responses_total  SLOs-probe_success
 
 *** Test Cases ***
-
 Test Existence of Prometheus Alerting Rules
-  [Tags]  Smoke  Sanity  ODS-509
-  Check Prometheus Alerting Rules
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-509
+    Check Prometheus Alerting Rules
 
 Test Existence of Prometheus Recording Rules
-  [Tags]  Smoke  Sanity  ODS-510
-  Check Prometheus Recording Rules
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-510
+    Check Prometheus Recording Rules
+
+Test Metric "Notebook CPU Usage" On ODS Prometheus
+    [Documentation]    Verifing the notebook cpu usage showing on RHODS promethues
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-178
+    ${cpu_usage_before} =    Read Current CPU Usage
+    Run Jupyter Notebook For 5 Minutes
+    ${cpu_usage_after} =    Read Current CPU Usage
+    Should Not Be Equal    ${cpu_usage_before}    ${cpu_usage_after}
+
+Test Metric "Rhods_Total_Users" On ODS Prometheus
+    [Documentation]    Verifies that metrics rhods_total_users and jupyterhub_total_users have the same value in Prometheus
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-628
+    ${expression} =    Set Variable    rhods_total_users&step=1  #step = 1 beacuase it will return value of current second
+    ${rhods_total_users} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    ${expression}
+    ${expression} =    Set Variable    jupyterhub_total_users&step=1  #step = 1 beacuase it will return value of current second
+    ${jupyterhub_total_users} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}
+    ...    ${expression}
+    Should Be Equal    ${rhods_total_users.json()["data"]["result"][0]["value"][-1]}
+    ...    ${jupyterhub_total_users.json()["data"]["result"][0]["value"][-1]}
+    ...    msg=metrics rhods_total_users and jupyterhub_total_users don't have the same value
+
+Test Metric "Rhods_Aggregate_Availability" On ODS Prometheus
```

Comment:
@rkamble5 could you rename this test to: `Test Metric "Rhods_Aggregate_Availability" On ODS Prometheus When There Are No Issues In Dashboard And JupyterHub`  ?

Reply:
done